tests/test_github.py

```diff
@@ -0,0 +1,746 @@
+                "url": "https://api.github.com/repos/zhquan_example/repo/issues/1",
+                "body": "Body",
+                "user_data": {
+                    "type": "User",
+                    "organizations_url": "https://api.github.com/users/zhquan_example/orgs",
+                    "repos_url": "https://api.github.com/users/zhquan_example/repos",
+                    "organizations": [
+                        {
+                            "login": "Orgs_1",
+                            "members_url": "https://api.github.com/orgs/Orgs_1/members{/member}",
+                            "avatar_url": "",
+                            "events_url": "https://api.github.com/orgs/Orgs_1/events",
+                            "hooks_url": "https://api.github.com/orgs/Orgs_1/hooks",
+                            "id": 1,
+                            "public_members_url": "https://api.github.com/orgs/Orgs_1/public_members{/member}",
+                            "repos_url": "https://api.github.com/orgs/Orgs_1/repos",
+                            "issues_url": "https://api.github.com/orgs/Orgs_1/issues",
+                            "url": "https://api.github.com/orgs/Orgs_1",
+                            "description": None
+                        },
+                        {
+                            "login": "Orgs_2",
+                            "members_url": "https://api.github.com/orgs/Orgs_2/members{/member}",
+                            "avatar_url": "",
+                            "events_url": "https://api.github.com/orgs/Orgs_2/events",
+                            "hooks_url": "https://api.github.com/orgs/Orgs_2/hooks",
+                            "id": 2,
+                            "public_members_url": "https://api.github.com/orgs/Orgs_2/public_members{/member}",
+                            "repos_url": "https://api.github.com/orgs/Orgs_2/repos",
+                            "issues_url": "https://api.github.com/orgs/Orgs_2/issues",
+                            "url": "https://api.github.com/orgs/Orgs_2",
+                            "description": None
+                        }
+                    ],
+                    "name": "zhquan_example",
+                    "url": "https://api.github.com/users/zhquan_example",
+                    "created_at": "2016-01-01T00:00:00Z",
+                    "updated_at": "2016-01-01T01:00:00Z",
+                    "login": "zhquan_example",
+                    "id": 1
+                },
+                "created_at": "2016-02-01T07:10:24Z",
+                "closed_at": "2016-02-01T12:13:21Z",
+                "id": 1
+            },
+            "perceval_version": "0.1.0",
+            "updated_on": 1454328801,
+            "origin": "https://github.com/zhquan_example/repo",
+            "uuid": "58c073fd2a388c44043b9cc197c73c5c540270ac",
+            "backend_version": "0.1.0"
+        }
+
+        for key in expected:
```

Comment:
Instead of using this `for` loop to check the values of the issue, you can use the method self.assertDictEqual to compare the two dictionaries.